src/components/Home.jsx

```diff
@@ -1,11 +1,128 @@
 import React from 'react';
-import { Grid } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+import { Button, Grid } from '@material-ui/core';
+import CallIcon from '@material-ui/icons/Call';
+import RestaurantIcon from '@material-ui/icons/Restaurant';
+import HomeIcon from '@material-ui/icons/Home';
+import HotTubIcon from '@material-ui/icons/HotTub';
+import BusIcon from '@material-ui/icons/DirectionsBus';
+import GroceryStoreIcon from '@material-ui/icons/LocalGroceryStore';
+import HealingIcon from '@material-ui/icons/Healing';
+import LibraryBooksIcon from '@material-ui/icons/LibraryBooks';
+import PetsIcon from '@material-ui/icons/Pets';
+import WifiIcon from '@material-ui/icons/Wifi';
+import WorkIcon from '@material-ui/icons/Work';
+import InfoIcon from '@material-ui/icons/Info';
```

Comment:
you could probably import these into an index file then require them from there...Like an interface. if you ever are required to switch away from material-ui icon library you'll find it challenging to find/replace every import.

Reply:
do you require some of these icons elsewhere in the codebase?

Reply:
Ya, good call. I'll split these out.